amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/domain/ParameterParsers.scala

```diff
@@ -433,19 +433,23 @@ case class Oas2ParameterParser(entryOrNode: YMapEntryLike,
     map.key(
       "schema",
       entry => {
-        OasTypeParser(entry, shape => setName(shape).asInstanceOf[Shape].adopted(payload.id))(toOas(ctx)) // i don't need to set param need in here. Its necesary only for form data, because of the properties
+        // i don't need to set param need in here. Its necessary only for form data, because of the properties
+        OasTypeParser(entry, shape => setName(shape).asInstanceOf[Shape].adopted(payload.id))(toOas(ctx))
           .parse()
           .map { schema =>
             checkNotFileInBody(schema)
             payload.set(PayloadModel.Schema, tracking(schema, payload.id), Annotations(entry))
             bindingRange.foreach { range =>
               schema.annotations += ParameterBindingInBodyLexicalInfo(range)
             }
-            schema
+            val result = schema
+            result
           }
       }
     )
-
+    if (payload.schema == null) {
```

Comment:
Can't this be done with an AMFRawValidation?

Reply:
we tried but it's not possible